hazelcast/src/test/java/com/hazelcast/nio/IOUtilTest.java

```diff
@@ -807,81 +858,120 @@ public void testGetPath_whenPathsInvalid() {
         getPath();
     }
 
-    @Test
-    public void testCopy_shouldDeepCopy() throws IOException {
-        File target = createFile("destination");
-        File source = createFile("source");
-        writeTo(source, "test content");
+    private File newFile(String filename) {
+        File file = new File(getFilename(filename));
+        files.add(file);
+        return file;
+    }
+
+    private File newFile(File parent, String filename) {
+        File file = new File(parent, getFilename(filename));
+        files.add(file);
+        return file;
+    }
+
+    private String getFilename(String filename) {
+        String name = "IOUtilTest-" + testName.getMethodName() + "-" + filename;
+        if (name.length() > 255) {
+            return name.substring(0, 255);
+        }
+        return name;
+    }
 
-        InputStream inputStream = new FileInputStream(source);
+    private File createFile(String fileName) {
+        return createFile(newFile(fileName));
+    }
+
+    private File createFile(File parent, String fileName) {
+        return createFile(newFile(parent, fileName));
+    }
+
+    private File createFile(File file) {
+        files.add(file);
+        if (file.isFile()) {
+            return file;
+        }
         try {
-            copy(inputStream, target);
-            assertTrue(equalContents(target, source));
-        } finally {
-            delete(target);
-            delete(source);
-            inputStream.close();
```

Comment:
Under Windows you cannot delete a file, while an `InputStream` on it is still open. Fixed this by closing the `InputStream` first.